src/index.ts

```diff
@@ -351,5 +351,15 @@ export class CdktfProviderProject extends cdk.JsiiProject {
       name: "Revert package.json version bump",
       run: "git checkout package.json",
     });
+
+    // Run copywrite tool to add copyright headers to all files
+    this.buildWorkflow?.addPostBuildSteps(
```

Comment:
Don't we need the config file in the pre-built provider repos as well?

Reply:
rrrrrrright 😅

Reply:
fixed!